mbed_lstools/lstools_base.py

```diff
@@ -403,6 +403,18 @@ def get_dummy_platform(self, platform_name):
         self.dummy_counter[platform_name] += 1
         return platform
 
+    def get_supported_platforms(self, device_type=None):
+        """! Return a dictionary of supported target ids and the corresponding platform name
+        @param device_type Filter which device entries are returned from the platform database
+        @return Dictionary of { 'target_id': 'platform_name', ... }
```

Comment:
is there some needs for target_id’s? Isn’t it enough for our tools to get just platform_names? (at least mbed-flasher does not need target-ids) function names says ”platforms” so at least I would expect to get just platform_names in array 🤔

Reply:
I mostly did it to not create two APIs for fetching platform names and target ids.

Reply:
That’s fine for me.